usaspending_api/accounts/views/common.py

```diff
@@ -0,0 +1,25 @@
+from usaspending_api.awards.serializers import FinancialAccountsByAwardsSerializer
```

Comment:
As you said it's kind of it's own thing, I'd rather leave it in common.py for now and move it when we give it a forever home. :)